pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestURLBuilder.java

```diff
@@ -110,8 +110,7 @@ public String forNewUpdateTableConfig(String tableName) {
   }
 
   public String forTableRebalance(String tableName, String tableType) {
-    String query = "rebalance?dryrun=false&type=" + tableType;
-    return StringUtil.join("/", _baseUrl, "tables", tableName, query);
+    return StringUtil.join("/", _baseUrl, "tables", tableName, "rebalance?type=" + tableType);
```

Comment:
I think that we should keep the one that gets all the parameters for rebalance api.

Reply:
Done